db/migrate/006_add_search_index_to_projects.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+Sequel.migration do
+  up do
+    run 'CREATE EXTENSION pg_trgm'
```

Comment:
Please, check for no exception if it is already exists. (This could be a case when extensions are pre-installed on DB).
Otherwise, you can add ```IF NOT EXISTS```